src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java

```diff
@@ -430,7 +448,17 @@ public static boolean containsGroup(JsonNode json, String group) {
   }
 
   // Create webhook for CI (using webhook proxy)
-  protected void createWebHooksForRepository(RepositoryData repo, OpenProjectData project) {
+  protected void createWebHooksForRepository(
+      RepositoryData repo, OpenProjectData project, String componentType) {
+
+    if (noWebhookComponents.contains(componentType)) {
```

Comment:
sharing with you my thought here... I'm not sure if this place is a good place for this `if (noWebhookComponents.contains(componentType))` condition or I would be better to place it in the caller function. Apart from this the implementation looks good to me. It is up to you if you prefer to move it to the caller function.

Reply:
Yes, I have chosen it this way because with the current implementation it was easier to test this way. Otherwise I would need to set up a mock party with a lot of interactions.